### PR TITLE
Add additional replace for | |

### PR DESCRIPTION
**Pull request by kelsonic**
## Description

This PR fixes an edge case where `" |  | "` is being sent from the CMS as a custom metatag, e.g. on the /resources page.

## Testing done
Updated unit test

## Screenshots
N/A

## Acceptance criteria
- [x] Fix double pipe edge case for title tags 

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/filters/liquid.js [hunk 1]
```diff
@@ -944,8 +944,9 @@ module.exports = function registerFilters() {
       formattedTitle = `${formattedTitle} | Veterans Affairs`;
     }
 
-    // Remove ' | | ' from the title.
+    // Remove ' | | ' and ' |  | ' from the title.
     formattedTitle = formattedTitle?.replace(' | | ', ' | ');
+    formattedTitle = formattedTitle?.replace(' |  | ', ' | ');
```
Comment (Rleahy22) [inline src/site/filters/liquid.js:949]:
```suggestion
    formattedTitle = formattedTitle?.replace(/\s*\|\s*\|/, ' | ');
```
Screen Shot 2021-07-15 at 11 06 46 AM

**Review by Rleahy22 — APPROVED** [review]

**Review by Rleahy22 — APPROVED** [review]
